### PR TITLE
feat(fair): implents fair articles landing screen

### PR DESCRIPTION
**Pull request by dzucconi**
The type of this PR is: **Feature**

This PR resolves [FX-2509]

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a [CHANGELOG.yml](/CHANGELOG.yml) entry or my changes do not require one.

[FX-2509]: https://artsyproduct.atlassian.net/browse/FX-2509

### CODE REVIEW

File: src/lib/Scenes/Fair2/Components/Fair2EditorialShare.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      onPress={() => {
+        Linking.openURL(
+          `https://twitter.com/intent/tweet?original_referer=${url}&amp;text=${subject}&amp;url=${url}&amp;via=artsy`
+        )
+      }}
+    >
+      <TwitterIcon title="Share this article on Twitter" />
+    </Touchable>
+  </Box>
+)
```
Comment (dzucconi) [inline src/lib/Scenes/Fair2/Components/Fair2EditorialShare.tsx:38]: Is this fine to just open URLs? Or is there some special share sheet thing we should be doing?

Comment (MounirDhahri) [reply src/lib/Scenes/Fair2/Components/Fair2EditorialShare.tsx:38]: yes this is the way 🎉

File: src/lib/Scenes/Fair2/Fair2Articles.tsx [hunk 2]
```diff
@@ -0,0 +1,204 @@
...
+              <Button
+                variant="secondaryGray"
+                block
+                width="100%"
+                loading={isLoading}
+                onPress={handlePress}
+                disabled={!relay.hasMore()}
+              >
+                Show more
+              </Button>
```
Comment (MounirDhahri) [inline src/lib/Scenes/Fair2/Fair2Articles.tsx:141]: **thought(non-blocking):** I think the previous block (96 -> 130 ) could have been inside a FlatList and the show more button can go inside `ListFooterComponent` maybe. You can check out `src/lib/Scenes/Sale/Components/SaleArtworkList.tsx`  for a similar implementation

File: src/lib/Scenes/Fair2/__tests__/Fair2Articles-tests.tsx [hunk 3]
```diff
@@ -0,0 +1,39 @@
+import { FAIR2_ARTICLES_QUERY, Fair2ArticlesPaginationContainer } from "lib/Scenes/Fair2/Fair2Articles"
+import { extractText } from "lib/tests/extractText"
+import { setupTestWrapper } from "lib/tests/setupTestWrapper"
+
+jest.unmock("react-relay")
+
+const { getWrapper } = setupTestWrapper({
```
Comment (MounirDhahri) [inline src/lib/Scenes/Fair2/__tests__/Fair2Articles-tests.tsx:7]: Just saw this now and it makes sense! 👏

Comment (MounirDhahri) [reply src/lib/Scenes/Fair2/__tests__/Fair2Articles-tests.tsx:7]: I will let the team know about it to start using it

**Review by MounirDhahri — APPROVED** [review]

**Comment by MounirDhahri** [comment]
Looks good to me 🎉🎉🎉. I added just one minor note that is not blocking about the usage of `Flatlist` to render the articles as a suggestion in case you missed it.

**Comment by dzucconi** [comment]
Thanks for the review @MounirDhahri! Updated to use `FlatList`

**Comment by MounirDhahri** [comment]
Thank you 🙏

**Pull request merged**